### PR TITLE
fix v2 issues

### PR DESCRIPTION
**Pull request by ferib**
- Bugfix webView.html missing
- Added `npm run compile-wsl` cuz Windows has no alias for `cp` 🙃

### CODE REVIEW

**Comment by kdssoftware** [comment]
Looks good 👌 lots of 'changes' xD

File: package.json [hunk 1]
```diff
@@ -1,132 +1,133 @@
...
-			{
-				"command": "lua.helper",
-				"title": "luaObfuscator | Advanced helper panel"
-			}
-		]
-	},
-	"scripts": {
-        "vscode:prepublish": "npm run compile",
-		"launch": "code --install-extension lua*.*.vsix && code .",
-		"compile": "cp src/utils/*.{json,html} dist/ && tsc",
```
Comment (kdssoftware) [inline package.json:105]: `"compile": "mkdir -p && cp src/utils/*.{json,html} dist/ && tsc",`

**Review by kdssoftware — CHANGES_REQUESTED** [review]
wont work without `mkdir`

**Comment by ferib** [comment]
> wont work without mkdir

it will, I left a "placeholder" file in the `dist` folder 🙃

**Comment by kdssoftware** [comment]
👌  placeholder is normally a `.gitkeep` tho. But we do want we want 🕺

**Pull request merged**